typespec-tests/src/main/java/com/parameters/spread/implementation/models/SpreadAsRequestBodyRequest1.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.parameters.spread.implementation.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * The SpreadAsRequestBodyRequest1 model.
+ */
+@Immutable
+public final class SpreadAsRequestBodyRequest1 implements JsonSerializable<SpreadAsRequestBodyRequest1> {
```

Comment:
Seems this model looks similar as [`SpreadAsRequestBodyRequest1`](https://github.com/Azure/autorest.java/blob/0cb0d48d8fa24ce8f7766cd7eea6d00b6fde04d8/typespec-tests/src/main/java/com/parameters/spread/implementation/models/SpreadAsRequestBodyRequest.java)?

Reply:
It is same. But they are from different operation (one in alias og, another in model og), so they are different model (even the content is same).

They probably also different entity in typespec.